modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase

```diff
@@ -107,4 +133,36 @@ definition {
 		}
 	}
 
+	@description = "LPS-155311. Assert the filter cannot be filtered by searching."
+	@priority = "3"
+	test FilterNotFound {
+		task ("When clicking on the filter button") {
+			FDSFilters.openFilters();
+		}
+
+		task ("And typing a key word in the category search box") {
+			FDSFilters.searchFilter(searchTerm = "Pending");
+		}
+
+		task ("And there is no matching filter") {
+			AssertElementNotPresent(
+				key_filter = "Pending",
+				locator1 = "FrontendDataSet#FILTER_OPTION");
+		}
+
+		task ("Then the user will be warned about it") {
+			AssertElementPresent(locator1 = "FrontendDataSet#FILTER_WARN_MESSAGE");
+		}
+	}
+
+	@description = "LPS-155303. Assert management bar will render."
+	@priority = "3"
+	test ManagementBarWillRender {
+		task ("Then the Management Bar will be displayed correctly") {
+			AssertElementPresent(locator1 = "FrontendDataSet#MANAGEMENT_BAR");
+
+			takeScreenshot();
```

Comment:
Hi @Esther-OC! Could you clarify how the Management Bar should be displayed correctly? Should we check for individual elements of the bar (filter, search, etc) or just that the bar is present is enough? Thanks!

cc: @chaowang-123

Reply:
Hi @susanchen3 & @chaowang-123!

I think it's enough to check that the Management Bar is present and displayed, as I guess that it's a component by itself, so its elements are already checked in their specific tests.